launchable/commands/record/commit.py

```diff
@@ -53,16 +53,30 @@ def commit(ctx, source: str, executable: bool, max_days: int, scrub_pii: bool, i
         _import_git_log(import_git_log_output, ctx.obj.dry_run)
         return
 
+    cwd = os.path.abspath(source)
     try:
-        exec_jar(os.path.abspath(source), max_days, ctx.obj.dry_run)
+        is_shallow = subprocess.check_output(
+            ['git', 'rev-parse', '--is-shallow-repository'],
+            stderr=subprocess.DEVNULL,
+            cwd=cwd,
+            universal_newlines=True).strip()
+        if is_shallow == "true":
+            click.echo(click.style(
+                "Can't collect all commits from {} since it is the shallow repository. "
+                "Please use full clone or disable commit collection by --no-commit-collection option."
```

Comment:
`use full clone` is a little strange for me.  `do full clone`...? but I'm not familiar with English...

Reply:
I referred use full clone from https://docs.aws.amazon.com/codepipeline/latest/userguide/tutorials-github-gitclone.html. What do you think?

Reply:
Nice, so you don't need to update it!